docs/compatibility.txt

```diff
@@ -0,0 +1,59 @@
+=============
+Compatibility
+=============
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+JDBC
+====
+
+This JDBC driver follows the JDBC 4.1 standard as it uses PGSQL driver as the
+underlying connector, however there are certain features that are not fully
+compatible with CrateDB.
+
+**The following features are not supported:**
+
+* ``java.sql.ParameterMetaData`` as returned by e.g. ``java.sql.PreparedStatement``
+* ``DataSource``
+* ``CallableStatement`` as CrateDB does not support stored procedures
+
+For further details about compatibility with all possible JDBC features,
+see the ``ResultSet``, ``ResultSetMetaData`` and ``DatabaseMetaData``
+implementations.
+
+**Though only an optional feature, it is still worth mentioning:**
+
+* the ``ResultSet`` is read only (``TYPE_FORWARD_ONLY``, ``CONCUR_READ_ONLY``),
+  so changes to a ``ResultSet`` are not supported. Though DDL and DML statements
+  are supported using the ``Statement`` and ``PreparedStatement`` interfaces.
+
+CrateDB
+=======
+
+All versions of the JDBC driver >= 2.0.x require CrateDB 0.56.0 or newer.
+Previous versions of the JDBC driver (1.x) are only compatible with older
+versions of CrateDB.
+
+The support for versions of the JDBC driver >= 2.0.x with CrateDB version
+0.56.x is **experimental**. The binding between versions of the JDBC
+driver >= 2.0.x and CrateDB will be stable when 0.57.x of CrateDB is stable.
```

Comment:
0.57.x is probably stable for a long time (and if it's not, we don't care about it now)

Reply:
fixed?